src/components/modal/configureOutputModal/MultiviewLayoutSettings/MultiviewLayout.tsx

```diff
@@ -0,0 +1,61 @@
+import { TList } from '../../../../hooks/useCreateInputArray';
+import { MultiviewViewsWithId } from '../../../../hooks/useSetupMultiviewLayout';
+import { MultiviewPreset } from '../../../../interfaces/preset';
+import Options from '../Options';
+
+export default function MultiviewLayout({
+  multiviewPresetLayout,
+  inputList,
+  handleChange
+}: {
+  multiviewPresetLayout: MultiviewPreset;
+  inputList: TList[] | undefined;
+  handleChange: (id: number, value: string) => void;
+}) {
+  return (
+    <div
+      className={`border-4 border-p/50 relative p-2 m-2`}
+      style={{
+        width: `${multiviewPresetLayout.layout.output_width}rem`,
+        height: `${multiviewPresetLayout.layout.output_height}rem`
+      }}
+    >
+      {(multiviewPresetLayout.layout.views as MultiviewViewsWithId[]).map(
```

Comment:
Do we need to force the type here? Maybe we should have a better typing for multiviewPresetLayout?